app/views/users/show.html.erb

```diff
@@ -0,0 +1,99 @@
+<%= render 'shared/menubar' %>
+<%= render 'shared/navbar' %>
+
+<% if @user == current_user %>
+  <div class="container text-center justify-content-center">
+    <h1>Hi, <%= current_user.first_name %>!</h1>
+    <div class="profile-avatar-box">
+      <% if @user.photo.attached? %>
+        <div class="profile-avatar" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('<%= cl_image_path @user.photo.key,crop: :fill %>')"></div>
+      <% else %>
+        <div class="profile-avatar" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('https://images.unsplash.com/photo-1519671282429-b44660ead0a7?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1770&q=80');"></div>
+      <% end %>
+    </div>
+    <p class="title">Superhost <i class="fas fa-concierge-bell"></i></p>
+    <p class="title"><i class="fa fa-star" aria-hidden="true"></i> Rating 4.5</p>
+  </div>
+
+
+  <div class="container">
+    <%= simple_form_for current_user, url: profile_path do |f| %>
+      <%= f.input :about %>
+      <%= f.input :first_name %>
+      <%= f.input :photo, as: :file, class: "photo-btn" %>
+      <%= f.submit "Save & Upload", class: "about-btn btn btn-default" %>
+    <% end %>
+  </div>
+
+  <div class="main m-5">
+    <h2>All events</h2>
+      <div class="overflow-auto mb-10 md:mb-20">
+        <div class="d-flex flex-row sm:space-y-0">
+
+          <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem">
+            <h3 class="profile-card mt-3 md:mt-0 i9prqz1">Events you created
+            </h3>
+          </div>
+
+          <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem" style="padding: 0px 10px;">
+            <h3 class="profile-card mt-3 md:mt-0 i9prqz1">Events you will attend
```

Comment:
Lets link it to bookings#index already

Reply:
`<%= link_to bookings_path do %>
            <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem" style="padding: 0px 10px;">
              <h3 class="profile-card pink mt-3 md:mt-0 i9prqz1">Events you will attend
              </h3>
            </div>
          <% end %>`